perseo-main/src/test/java/com/telefonica/iot/perseo/test/Help.java

```diff
@@ -39,46 +42,70 @@
 
     public static final int PORT = 8129;
 
-    public static final String[] ExampleRules = new String[]{
-        "select id, price? as Price from iotEvent.win:length(100) group by id",
-        "@Audit select *,\"blood_1_action\" as iotcepaction,"
-        + "ev.BloodPressure? as Pression, ev.id? as Meter from pattern "
-        + "[every ev=iotEvent(cast(cast(BloodPressure?,String),float)>1.5"
-        + " and type=\"BloodMeter\")]"};
-    public static final String[] ExampleNotices = new String[]{
-        "{\n"
-        + "\"BloodPressure\": 2,\n"
-        + "\"id\":\"guay!\",\n"
-        + "\"otro\":\"mas\",\n"
-        + "\"numero\":4,\n"
-        + "\"sub\": {\n"
-        + "	\"subnumero\":18,\n"
-        + "	\"subcadena\":\"SUB2\",\n"
-        + "	\"subflotante\": 12.3,\n"
-        + "	\"sub2\": { \"valor\": 3}\n"
-        + "	}\n"
-        + "}"};
-
+    public static String[] ExampleRules() {
+        return new String[]{
+                 "select id, price? as Price from iotEvent.win:length(100) group by id",
+                    "@Audit select *,\"blood_1_action\" as iotcepaction,"
+                    + "ev.BloodPressure? as Pression, ev.id? as Meter from pattern "
+                    + "[every ev=iotEvent(cast(cast(BloodPressure?,String),float)>1.5"
+                    + " and type=\"BloodMeter\")]"};  
+    }
+    public static String[] ExampleNotices() {
+        return new String[]{
+                "{\n"
+                        + "\"BloodPressure\": 2,\n"
+                        + "\"id\":\"guay!\",\n"
+                        + "\"otro\":\"mas\",\n"
+                        + "\"numero\":4,\n"
+                        + "\"sub\": {\n"
+                        + " \"subnumero\":18,\n"
+                        + " \"subcadena\":\"SUB2\",\n"
+                        + " \"subflotante\": 12.3,\n"
+                        + " \"sub2\": { \"valor\": 3}\n"
+                        + " }\n"
+                        + "}"};
```

Comment:
This indent should look like more like the origina one, e.g. see the aligment of `"{\n"` and `+ "}"};`

Reply:
Fixed in b6d7148bbc37f5c8a5de01bf50c6c12a6d80d237